synapse/types.py

```diff
@@ -59,9 +59,11 @@
 StateMap = Mapping[StateKey, T]
 MutableStateMap = MutableMapping[StateKey, T]
 
-# the type of a JSON-serialisable dict. This could be made stronger, but it will
-# do for now.
+# JSON types. These could be made stronger, but will do for now.
+# A JSON-serialisable dict.
 JsonDict = Dict[str, Any]
+# A JSON-serialisable object.
+JsonObject = object
```

Comment:
This surprised me. I guessed that "JsonObject" and "JsonDict" would be one and the same, because javascript/json calls its mapping types objects (c.f. json.org):

> An object is an unordered set of name/value pairs. An object begins with {left brace and ends with }right brace. Each name is followed by :colon and the name/value pairs are separated by ,comma.

Perhaps `JsonSerializable`?

Reply:
Ah, bad naming on my part. I was thinking `object`, as in the class that's at the base of the type hierarchy.

Reply:
+1 that I found this confusing a JSON object == a dict in Python!

Reply:
Renamed it!